include/ensmallen_bits/nsga2/nsga2_impl.hpp

```diff
@@ -423,9 +423,9 @@ inline bool NSGA2::Dominates(
 //! Assign crowding distance to the population.
 template <typename MatType>
 inline void NSGA2::CrowdingDistanceAssignment(
-	const std::vector<size_t>& front,
-	std::vector<arma::Col<typename MatType::elem_type>>& calculatedObjectives,
-	std::vector<typename MatType::elem_type>& crowdingDistance)
+    const std::vector<size_t>& front,
+    std::vector<arma::Col<typename MatType::elem_type>>& calculatedObjectives,
+    std::vector<typename MatType::elem_type>& crowdingDistance)
```

Comment:
these are just changes in whitespace. what's the reason for this?  if the changes are not related to Active CMAES, please omit them

Reply:
It fixes the style to only indent twice on a wrapped line---personally I think it's just fine.

Reply:
The whitespace changes are not a problem by themselves.  The issue is that this PR seems to be touching stuff that's not directly related to it.  I'm worried that there may be other irrelevant files that are also changed, possibly introducing regressions.

Reply:
These changes were pushed with the last PR [351](https://github.com/mlpack/ensmallen/pull/367).

Reply:
Thanks